sdk/ml/azure-ai-ml/tests/workspace/unittests/test_workspace_operations_base.py

```diff
@@ -349,12 +349,13 @@ def test_populate_feature_store_arm_parameters(
         )
 
         # test create feature store
-        feature_store = FeatureStore(name="name", resource_group="rg")
+        feature_store = FeatureStore(name="name", resource_group="rg", location="eastus2euap")
         template, param, _ = mock_workspace_operation_base._populate_arm_parameters(
             workspace=feature_store, grant_materialization_permissions=True
         )
 
         assert param["kind"]["value"] == "featurestore"
+        assert param["location"]["value"] == "eastus2euap"
```

Comment:
do we want to use canary as the default value?

Reply:
this is just for unit tests. The default value is always workspace location: user provided location, if not then resource source group location.